src/components/ratings_reviews/ReviewListEntry.jsx

```diff
@@ -80,32 +90,66 @@ function ReviewListEntry({ review, submitHelpfulNess }) {
           <ReviewPhotos photo={photo} key={photo.id} />
         ))}
       </div>
-      <div
-        className="review-char-container"
-        style={{
-          display: 'flex',
-          justifyContent: 'space-between',
-          width: '65%',
-        }}
-      >
-        <span>Size: Awesome </span>
-        <span>Fits: True to Size</span>
-        <span>Length: Long</span>
-      </div>
 
       <div
         className="reviewer-name"
         style={{
           display: 'flex',
           width: '100%',
           color: 'darkgray',
-          margin: '2em 0em 1em 0.2em',
+          margin: '2em 0em 0em 0.2em',
+          fontSize: '0.8em',
         }}
       >
         {review.reviewer_name} |{' '}
-        {format(parseISO(review.date), 'MMMM dd, yyyy')} | Verified User |{' '}
-        {review.recommend && 'Recommended!'}
+        {format(parseISO(review.date), 'MMMM dd, yyyy')} | Verified User{' '}
+        <FontAwesomeIcon
+          icon={faCertificate}
+          size="xs"
+          style={{
+            opacity: '0.5',
+            margin: '0.3em 0.2em',
+          }}
+        />
       </div>
+      <span
+        style={{
+          fontSize: '0.65em',
+          margin: '0.7rem 0rem',
+        }}
+      >
+        {review.recommend && (
+          <div>
+            <FontAwesomeIcon
+              icon={faCircleCheck}
+              size="XL"
```

Comment:
Quick fix from console:

_Failed prop type: Invalid prop `size` of value `XL` supplied to `FontAwesomeIcon`, expected one of ["2xs","xs","sm","lg","xl","2xl","1x","2x","3x","4x","5x","6x","7x","8x","9x","10x"]._